rust/operator-binary/src/controller.rs

```diff
@@ -1161,3 +1027,114 @@ fn liveness_probe(trino: &TrinoCluster) -> Probe {
         ..Probe::default()
     }
 }
+
+fn create_tls_volume(volume_name: &str, tls_secret_class: Option<&TlsSecretClass>) -> Volume {
```

Comment:
Why does this take an `Option<&TlsSecretClass>`? That pushes the caller into the following pattern:

```rust
let tls = trino.client_tls();
if tls.is_some() {
  // do other prep
  create_tls_volume(tls);
}
```

I think it would make more sense for this to take a `&TlsSecretClass` instead, and having the caller look like this:

```rust
if let Some(tls) = trino.client_tls() {
  // do other prep
  create_tls_volume(tls);
}
```

Reply:
done